lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_eager_fallback.cpp

```diff
@@ -55,8 +55,12 @@ void ltc_eager_fallback(const c10::OperatorHandle& op,
                  torch::lazy::getBackend()->EagerFallbackDeviceType());
 }
 
+std::function<void(void)> register_ts_ltc_eager_fallback;
+
 TORCH_LIBRARY_IMPL(_, Lazy, m) {
-  m.fallback(torch::CppFunction::makeFromBoxedFunction<&ltc_eager_fallback>());
+  register_ts_ltc_eager_fallback = [&]() {
+    m.fallback(torch::CppFunction::makeFromBoxedFunction<&ltc_eager_fallback>());
```

Comment:
this one doesn't need to be guarded under EAGER_REGISTRATION=1?

It's currently in a file that is only part of the TS backend, so that works for now.   I think we may want to share some of the fallback logic with xla / other backends later, but i guess we can just update the registration at that time.

Reply:
Its guarded under `EAGER_REGISTRATION=1` as a different LTC backend may want to implement different fallback logic. Which wouldn't be possible if the TS backend registers the fallback at c++ static initialization time.